tests/proofs/dexter/dexter-spec.md

```diff
@@ -15,6 +15,80 @@ module DEXTER-SPEC
 
   claim <k> (now 0 => .) ~> #dexterCode(_) ... </k>
         <mynow> #Timestamp(0) </mynow>
+
+  claim <k> PUSH nat X:Int ; PUSH nat Y:Int ; PUSH nat Z:Int ; MUL ; EDIV ; IF_NONE {} { DUP ; CDR ; SWAP ; CAR ; PUSH nat 0 ; DIG 2 ; COMPARE ; EQ } => . ... </k>
+        <stack> .Stack => [ bool ?_ ] ; [ nat (Z *Int Y) /Int X ] ; .Stack </stack>
+    requires X >Int 0
+     andBool Y >=Int 0
+     andBool Z >=Int 0
+```
+
+```k
+endmodule
+```
+
+## Add Liquidity
+
+Allows callers to "mint" liquidity in excahnge for tez.
+Caller gains liquidity equal to `tez * storage.lqtTotal / storage.xtzPool`.
+
+-   Storage updates:
+
+```
+Storage( lqtTotal:  LqtTotal  => LqtTotal  + lqt_minted ;
+         tokenPool: TokenPool => TokenPool + tokens_deposited ;
+         xtzPool:   XtzPool   => XtzPool   + Tezos.amount
+       )
+```
+
+-   Operations:
+
+1. call to `transfer` entrypoint of liquidity address: Send tokens from sender to self.
+2. call to `mintOrBurn` entrypoint of token address: Adds liquidity for the sender.
+
+-   Preconditions
+
+1.  the token pool is _not_ currently updating (i.e. `storage.selfIsUpdatingTokenPool = false`)
+2.  the deadline has not passed (i.e. the `Tezos.now < input.deadline`)
+3.  the tez transferred is less than `input.maxTokensDeposited`
```

Comment:
it should be "token" transferred, not "tez" transferred here.

Reply:
thanks, thought I fixed that, must have undone accidentally